src/Microsoft.VisualStudio.Editors/XmlIntellisense/XmlIntellisense.vb

```diff
@@ -105,7 +105,7 @@ Namespace Microsoft.VisualStudio.Editors.XmlIntellisense
     '   Methods on this class should only be called on the VS foreground
     '   thread.
     '--------------------------------------------------------------------------
-    <ClassInterface(ClassInterfaceType.None)> _
+    <ClassInterface(ClassInterfaceType.None)>
```

Comment:
It's interesting that this seems to be pretty-listing the whole file.  @jinujoseph potential bug for you here.